docs/triggerbindings.md

```diff
@@ -13,13 +13,51 @@ metadata:
 spec:
   params:
   - name: gitrevision
-    value: $(event.head_commit.id)
+    value: $(body.head_commit.id)
   - name: gitrepositoryurl
-    value: $(event.repository.url)
+    value: $(body.repository.url)
+  - name: contenttype
+    value: $(header.Content-Type)
 ```
 
 `TriggerBinding`s are connected to `TriggerTemplate`s within an [`EventListener`](eventlisteners.md), which is where the pod is actually instantiated that "listens" for the respective events.
 
 ## Parameters
 `TriggerBinding`s can provide `params` which are passed to a
 `TriggerTemplate`. Each parameter has a `name` and a `value`.
+
+
+## Event Variable Interpolation
+
+### Body
```

Comment:
Nice docs! Quick question: Does it work with arrays e.g. can I do $(body.key[1].key2)
Also, is this syntax different from JSONPath?

Reply:
Thanks for taking a look @dibyom!
I haven't changed the event/body path functionality, so we currently can't do `$(body.key[1].key2)`. We're using the [gjson](https://github.com/tidwall/gjson) library, so we could technically do anything supported by gjson, however, right now we restrict body paths to [this regex](https://github.com/tektoncd/triggers/blob/master/pkg/template/event.go#L30) `\$\(body(.[0-9A-Za-z_-]+)*\)`.

Reply:
I don't think that gjson uses the JSONPath syntax though.

Reply:
cool cool! as long as it is some well known syntax :+1:

Reply:
At the moment, I don't think that we should advertise the gjson syntax, because we currently don't allow users to use the full gjson syntax. We only allow `$(body.X.Y.Z)` right now.